integrations/qdrant/src/haystack_integrations/document_stores/qdrant/converters.py

```diff
@@ -19,7 +19,15 @@ def documents_to_batch(
         points = []
         for document in documents:
             payload = document.to_dict(flatten=False)
-            vector = payload.pop(embedding_field) or {}
+            vector = {}
+            if embedding_field in payload and payload[embedding_field] is not None:
+                dense_vector = payload.pop(embedding_field) or []
+                vector["text-dense"] = dense_vector
+            # TODO: Adapt to Haystack Modification of the Document Dataclass
+            if "_sparse_vector" in payload["meta"]:
+                sparse_vector = payload["meta"].pop("_sparse_vector", {"indices": [], "values": []})
+                sparse_vector_instance = rest.SparseVector(**sparse_vector)
+                vector["text-sparse"] = sparse_vector_instance
```

Comment:
before haystack document class modification, we have to store it in the meta field